src/proto/streams/prioritize.rs

```diff
@@ -175,6 +189,8 @@ impl Prioritize {
             self.try_assign_capacity(stream);
         }
 
+        self.current_send_buffer_size += sz as usize;
```

Comment:
I used to do this prior to calling `try_assign_capacity`. AFAIU that was wrong, right?

Reply:
That sounds right. The connection-level and stream-level should be updated together, but since updating the stream-level may make it exceed it's requested capacity and thus needs to check for more capacity, that must happen in-between.

A code comment about the ordering being important may be prudent.